django/db/backends/base/schema.py

```diff
@@ -593,7 +593,11 @@ def _alter_field(self, model, old_field, new_field, old_type, new_type,
                 ))
         # Nullability change?
         if old_field.null != new_field.null:
-            if new_field.null:
+            if (self.connection.features.interprets_empty_strings_as_nulls and
+                new_field.get_internal_type() in ("CharField", "TextField")):
```

Comment:
The recommendation is to double indent in this particular case so as to avoid visual confusion with the contents of the block.